dashboard/modules/metrics/metrics_head.py

```diff
@@ -249,19 +255,14 @@ def _create_default_grafana_configs(self):
             "w",
         ) as f:
             f.write(GRAFANA_DATASOURCE_TEMPLATE.format(prometheus_host=prometheus_host))
-
-        # Output the dashboards in a special directory
-        if self._grafana_dashboard_output_dir:
-            grafana_dashboards_dir = os.path.join(
-                GRAFANA_CONFIG_INPUT_PATH, "dashboards"
-            )
-            # Copy all dashboard jsons from directory
-            for root, _, files in os.walk(grafana_dashboards_dir):
-                for file in files:
-                    shutil.copy2(
-                        os.path.join(root, file),
-                        os.path.join(self._grafana_dashboard_output_dir, file),
-                    )
+        with open(
+            os.path.join(
+                dashboards_path,
+                "default_grafana_dashboard.json",
+            ),
```

Comment:
This is the issue, the output path should be `_grafana_dashboard_output_dir`. That value can be overwritten via ENV VAR.